notebooks/bnn_hierarchical_blackjax.ipynb

```diff
@@ -35,7 +35,7 @@
       "source": [
         "# Hierarchical Bayesian neural networks\n",
         "\n",
```

Comment:
This commit is not doing anything, and I don't see any difference with the notebook that is already on master. Are you sure you opened a PR with the right branch?

Reply:
You are right, it was the wrong version. I've updated it.